src/core/attribute.d

```diff
@@ -0,0 +1,64 @@
+/**
+ * This module contains UDA's (User Defined Attributes) either used in
+ * the runtime or special UDA's recognized by compiler.
+ *
+ * Copyright: Copyright Jacob Carlborg 2015.
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   Jacob Carlborg
+ * Source:    $(DRUNTIMESRC core/_attribute.d)
+ */
+
+/*          Copyright Jacob Carlborg 2015.
+ * Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+ */
+module core.attribute;
+
+/**
+ * This enum represent a meta attribute. Any declaration that has this attribute
+ * attached to itself is to be considered an attribute. That declaration
+ * should only be used as an attribute and never on its own.
+ */
+enum attribute;
```

Comment:
There is no need to "identify attribute definitions in metadata" because anything that `__traits(getAttributes, Type)` returns is an attribute. What's the difference between an attributed attribute and a normal attribute?